Jenkinsfile

```diff
@@ -93,29 +78,37 @@ pipeline {
       steps {
         sh "sed 's/__IMAGE_TAG__/${GIT_COMMIT}/g' kubernetes/deployment-staging.tmpl | kubectl --context azure apply --record -f -"
       }
-      post {
-        success {
-          script {
-            if (env.BRANCH_NAME == 'master') {
-              slackSend (
-                color: '#00FF00',
-                message: "SUCCESSFUL: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
-                channel: "#ops"
-              )
-            }
-          }
+    }
+
+    stage('Migrate staging database') {
```

Comment:
this will have to go before the staging deploy or combine into one staging stage with a step to migrate and a step to deploy.